tests/linearizability/model.go

```diff
@@ -17,33 +17,64 @@ package linearizability
 import (
 	"encoding/json"
 	"fmt"
+	"strings"
 
 	"github.com/anishathalye/porcupine"
 )
 
-type Operation string
+type OperationType string
 
 const (
-	Get    Operation = "get"
-	Put    Operation = "put"
-	Delete Operation = "delete"
-	Txn    Operation = "txn"
+	Get    OperationType = "get"
+	Put    OperationType = "put"
+	Delete OperationType = "delete"
+	Txn    OperationType = "txn"
 )
 
 type EtcdRequest struct {
-	Op            Operation
+	Conds []EtcdCondition
+	Ops   []EtcdOperation
```

Comment:
can you please comment on when to expect multiple operations in a request? what is the main use case? thanks

Reply:
Multi operational transactions. KV requests like put, get can be also modeled as transactions with one operation.

Reply:
[note: k8s uses only one operation/put per transaction, but etcd supports multiple]